src/main/java/org/fundacionjala/coding/dennis/Customer.java

```diff
@@ -0,0 +1,93 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ *
+ */
+public class Customer {
+    private String name;
+    private Vector rentals = new Vector(1, 1);
+
+    /**
+     *
+     * @param name asigned.
+     */
+    public Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     *
+     * @param arg add rental.
+     */
+    public void addRental(final Rental arg) {
+        rentals.addElement(arg);
+    }
+
+    /**
+     *
+     * @return how many rentals.
+     */
+    public int numRentalElements() {
+        return rentals.size();
+    }
+
+    /**
+     *
+     * @return how big is the bag.
+     */
+    public int capacityRentalElements() {
+        return rentals.capacity();
+    }
+
+    /**
+     *
+     * @return give me the name.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Calculus of data.
+     */
+    public void statement() {
+        float totalAmount = 0.0f;
+        float thisPrice;
+        int frequentRenterPoints = 0;
+        int thisFrequentRenterPoints;
+        Enumeration rentals = this.rentals.elements();
+        System.out.println(customerName());
+        while (rentals.hasMoreElements()) {
+            Rental each = (Rental) rentals.nextElement();
+            thisPrice = each.calculatePriceMovie();
+            System.out.println(movieAndPrice(each, thisPrice));
+            thisFrequentRenterPoints = each.calculateFrequentRenterPoints();
+            frequentRenterPoints = frequentRenterPoints + thisFrequentRenterPoints;
+            totalAmount += thisPrice;
+        }
+        System.out.println("Amount owed is ".concat(String.valueOf(totalAmount)));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)